src/frontend/calendar/index.js

```diff
@@ -64,7 +65,7 @@ function Calendar() {
 				</nav>
 				<div>
 					<h2 aria-live="polite" aria-atomic>
-						{ date( 'F Y', new Date( year, month, 1 ) ) }
```

Comment:
There was a suggestion on [the issue](https://github.com/WordPress/meeting-calendar/issues/130#issuecomment-1135717404) to use `2` here instead, so that even if the date converts to before midnight, it will still be the first of the month. That feels like a more straightforward workaround to me, what do you think?

Reply:
I've considered just fixing the issue with using `2` here originally. But it seems not to explain too much by code itself, so I thought it would be nice to adjust the date with its offset.
But on the other hand, I think it's also making sense to just use `2` here as a workaround in regard to `real offset`, `reverse offset` difference and the fact that it brings into more stuff that needs a developer to figure out.

I am ok with changing it to using `2` here, and I'll add some comments here.
What do you think? @ryelle @StevenDufresne